dev_guide/managing_images.adoc

```diff
@@ -491,20 +493,23 @@ every time a build starts. This forces an image access check on every build,
 removing the possibility of users accessing the build node's local cache of an
 image they do not have permission to access.
 ====
-
+endif::openshift-enterprise,openshift-origin[]
+ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
 If a container's `*imagePullPolicy*`
 parameter is not specified, {product-title} sets it based on the image's tag:
 
 . If the tag is *latest*, {product-title} defaults `*imagePullPolicy*` to `*Always*`.
 . Otherwise, {product-title} defaults `*imagePullPolicy*` to `*IfNotPresent*`.
 endif::[]
```

Comment:
Should this `endif` have the same matching

> openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated

from above?

Reply:
Actually, it looks superfluous...could remove this and then the one you added on line 505

Reply:
Otherwise, LGTM 👍🏻

Reply:
@wgordon17 Thanks for looking! I removed the `ifdef` from 505.